postprocessing/comstockpostproc/resources/comstock_column_definitions.csv

```diff
@@ -1611,12 +1611,227 @@ calculated,out.utility_bills.electricity_bill_intensity,out.utility_bills.electr
+calculated,calc.percent_savings.emissions.intensity.electricity.shoulder_daily_average.lrmer_high_re_cost_15_2023_start,calc.percent_savings.emissions.intensity.electricity.shoulder_daily_average.lrmer_high_re_cost_15_2023_start,FALSE,FALSE,float,percent,percent,"Percent savings of daily average greenhouse gas emissions per floor area during shoulder season from electricity use, using Cambium 2022 Long-Range Marginal Emissions Rate high renewable energy cost emissions intensity values levelized over 15 years starting in 2023"
+calculated,calc.percent_savings.emissions.intensity.electricity.shoulder_daily_average.lrmer_low_re_cost_15_2023_start,calc.percent_savings.emissions.intensity.electricity.shoulder_daily_average.lrmer_low_re_cost_15_2023_start,FALSE,FALSE,float,percent,percent,"Percent savings of daily average greenhouse gas emissions per floor area during shoulder season from electricity use, using Cambium 2022 Long-Range Marginal Emissions Rate low renewable energy cost emissions intensity values levelized over 15 years starting in 2023"
+calculated,calc.percent_savings.emissions.intensity.electricity.shoulder_daily_average.lrmer_mid_case_15_2023_start,calc.percent_savings.emissions.intensity.electricity.shoulder_daily_average.lrmer_mid_case_15_2023_start,FALSE,FALSE,float,percent,percent,"Percent savings of daily average greenhouse gas emissions per floor area during shoulder season from electricity use, using Cambium 2022 Long-Range Marginal Emissions Rate mid case emissions intensity values levelized over 15 years starting in 2023"
 calculated,calc.percent_savings.electricity.utility_bills.electricity_bill_median_intensity,calc.percent_savings.electricity.utility_bills.electricity_bill_median_intensity,TRUE,FALSE,float,percent,percent,Median electric bill per floor area
 calculated,calc.percent_savings.electricity.utility_bills.electricity_bill_mean_intensity,calc.percent_savings.electricity.utility_bills.electricity_bill_mean_intensity,TRUE,FALSE,float,percent,percent,Mean electric bill per floor area
 calculated,calc.percent_savings.electricity.utility_bills.electricity_bill_min_intensity,calc.percent_savings.electricity.utility_bills.electricity_bill_min_intensity,TRUE,FALSE,float,percent,percent,Minimum electric bill per floor area
 calculated,calc.percent_savings.electricity.utility_bills.electricity_bill_max_intensity,calc.percent_savings.electricity.utility_bills.electricity_bill_max_intensity,TRUE,FALSE,float,percent,percent,Max electric bill per floor area
-calculated,out.utility_bills.electricity_bill_max_intensity..usd_per_ft2,out.utility_bills.electricity_bill_max_intensity..usd_per_ft2,TRUE,FALSE,float,percent,percent,Max total electricity bill intensity across all electricity rates in US dollars per square foot
```

Comment:
I believe this is a duplicated row (with extra string) with line 1835? @asparke2